vocode/streaming/transcriber/google_transcriber.py

```diff
@@ -9,14 +9,14 @@
 
 from vocode.streaming.models.audio_encoding import AudioEncoding
 from vocode.streaming.transcriber.base_transcriber import (
-    BaseTranscriber,
+    BaseThreadAsyncTranscriber,
     Transcription,
 )
 from vocode.streaming.models.transcriber import GoogleTranscriberConfig
 from vocode.streaming.utils import create_loop_in_thread
 
 
-class GoogleTranscriber(BaseTranscriber):
+class GoogleTranscriber(BaseThreadAsyncTranscriber):
```

Comment:
I believe you should be able to make the Google synth totally async by using TextToSpeechAsyncClient.

Reply:
ah, leaving as a TODO